### PR TITLE
Repair Activate Method 

### PR DESCRIPTION
**Pull request by lucasBertola**
Repair Activate Method.

#78

### CODE REVIEW

**Comment by lucasBertola** [comment]
up :)

**Comment by lucasBertola** [comment]
up :)

File: test/synaptic.js [hunk 1]
```diff
@@ -430,36 +430,50 @@ describe("LSTM - Timing Task", function() {
 });
 
 describe("Optimized and Unoptimized Networks Equivalency", function() {
-  var optimized = new LSTM(2,1,1)
+ var optimized = new LSTM(1,1,1)
```
Comment (cazala) [inline test/synaptic.js]: Why did you remove an input?

File: src/neuron.js [hunk 2]
```diff
@@ -46,16 +46,21 @@ Neuron.prototype = {
...
-
     // eq. 15
-    this.state = this.selfconnection.gain * this.selfconnection.weight *
-      this.state + this.bias;
+    this.state = this.selfconnection.gain * this.selfconnection.weight * this.state + this.bias;
 
     for (var i in this.connections.inputs) {
       var input = this.connections.inputs[i];
-      this.state += input.from.activation * input.weight * input.gain;
+      if(input.gater){
```
Comment (cazala) [inline src/neuron.js]: I don't see what you are achieving here, if an input is a not a gater, the `gain` is always `1`, so there's no need for an if statement here and the second for loop below on line 58
